packages/expo-file-system/src/FileSystem.types.ts

```diff
@@ -0,0 +1,63 @@
+export type DownloadOptions = {
+  md5?: boolean;
+  headers?: { [key: string]: string };
+};
+
+export type DownloadResult = {
+  uri: string;
+  status: number;
+  headers: { [key: string]: string };
+  md5?: string;
+};
+
+export type DownloadProgressCallback = (data: DownloadProgressData) => void;
+
+export type DownloadProgressData = {
+  totalBytesWritten: number;
+  totalBytesExpectedToWrite: number;
+};
+
+export type DownloadPauseState = {
+  url: string;
+  fileUri: string;
+  options: DownloadOptions;
+  resumeData?: string;
+};
+
+export type FileInfo =
+  | {
+      exists: true;
+      uri: string;
+      size: number;
+      modificationTime: number;
+      md5?: string;
+    }
+  | {
+      exists: false;
+      isDirectory: false;
+    };
+
+export enum EncodingTypes {
```

Comment:
```suggestion
export enum EncodingType {
```

Use singular for enum types where you'd use just one option at a time (ex: a file can be UTF-8 or Base64 but not both) and plural when you expect to combine options as if they were bit masks (ex: file is readable and writable).